setup.py

```diff
@@ -26,6 +26,7 @@
 PYTHON_REQUIRES = ">=3.7"
 REQUIRES = [
     "urllib3 >= 1.25.3, < 2.1.0",
+    "httpx",
```

Comment:
httpx is used by our custom `token.py` so is not added as a dependency by the autogeneration, and needs to be reintroduced each time.